acceptance/suites/tests/authorization/hup_reloads_crl.rb

```diff
@@ -0,0 +1,68 @@
+test_name '(SERVER-1380) CRL should reload after a server HUP' do
```

Comment:
It seems like all the steps in this test will only run if there's an agent (not on the master) in addition to a master. Would it be possible/worthwhile to add something to the top of this file saying "skip this test unless there's an additional agent"? It seems like having it be explicit that that's the only scenario in which this test will run might good.